examples/deployment/new-api.ts

```diff
@@ -22,48 +22,50 @@ async function main() {
       networkOwnerId: "test",
     })
     .createActivityPool("app", {
-      image: "file://golem/node:20",
-      // image: "golem/node:20",
-      // image: "http://golem.io/node:20",
-      // imageHash: "0x30984084039480493840",
-      resources: {
-        minCpu: 4,
-        minMemGib: 8,
-        minStorageGib: 16,
-      },
-      replicas: 10,
-      network: "basic",
-      market: {
-        rentHours: 12 /* REQUIRED */,
-        pricing: {
-          maxStartPrice: 1 /* REQUIRED */,
-          maxCpuPerHourPrice: 1 /* REQUIRED */,
-          maxEnvPerHourPrice: 1 /* REQUIRED */,
+      demand: {
```

Comment:
Yeah, let's discuss this a bit, the second argument shouldn't need that `demand` here. Try to propose different types that will satisfy this requirement.